packages/playground/_stories/main/Button/Button.stories.ts

```diff
@@ -7,87 +9,79 @@ import type { UI5StoryArgs } from "../../../types.js";
 
 import { DocsPage } from "../../../.storybook/docs";
 
-// @ts-ignore
 import type Button from "@ui5/webcomponents/dist/Button.js";
+import ButtonDesign from "@ui5/webcomponents/dist/types/ButtonDesign.js";
 
 const component = "ui5-button";
 
 export default {
-    title: "Main/Button",
-    component,
-    parameters: {
-        docs: {
-          page: DocsPage({ ...componentInfo, component })
-        },
-    },
-    argTypes,
+	title: "Main/Button",
+	component,
+	parameters: {
+		docs: {
+			page: DocsPage({ ...componentInfo, component })
+		},
+	},
+	argTypes,
 } as Meta<Button>;
 
-const Template: UI5StoryArgs<Button, StoryArgsSlots> = (args) => html`<div></div>`;
+const Template: UI5StoryArgs<Button, StoryArgsSlots> = (args) => html`
+<ui5-button
+	design="${ifDefined(args.design)}"
+	?disabled="${ifDefined(args.disabled)}"
+	icon="${ifDefined(args.icon)}"
+	?icon-end="${ifDefined(args.iconEnd)}"
+	tooltip="${ifDefined(args.tooltip)}"
+	accessible-name="${ifDefined(args.accessibleName)}"
+	accessible-name-ref="${ifDefined(args.accessibleNameRef)}"
+	accessibility-attributes="${ifDefined(args.accessibilityAttributes)}"
+>
+	${unsafeHTML(args.default)}
+</ui5-button>`;
 
+export const Basic = Template.bind({});
+Basic.args = {
+	default: "Button",
+	design: ButtonDesign.Default,
+};
 
-export const Template0: StoryFn = () => html`
-<h3>Basic Button</h3>
-	<div class="snippet">
-		<ui5-button class="samples-margin" design="Default">Default</ui5-button>
-		<ui5-button class="samples-margin" disabled="">Disabled</ui5-button>
-		<ui5-button class="samples-margin" design="Transparent">Cancel</ui5-button>
-		<ui5-button class="samples-margin" design="Positive">Approve</ui5-button>
-		<ui5-button class="samples-margin" design="Negative">Decline</ui5-button>
-		<ui5-button class="samples-margin" design="Attention">Warning</ui5-button>
-		<ui5-button class="samples-margin" design="Emphasized">Subscribe</ui5-button>
-	</div>
-`;
+export const Design = Template.bind({});
+Design.args = {
```

Comment:
not sure if we need different sample for design as it can be done with dropdown in prev sample